### PR TITLE
update gen_pages.sh

### PR DESCRIPTION
**Pull request by bookshelfdave**
running `cd util && ./gen_pages.sh`
will generate docs here: http://basho.github.io/riak-c-client/index.html

### CODE REVIEW

File: util/gen_pages.sh [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
+cwd=${PWD##*/}
+if [ "$cwd" != "util" ]
+then
+  echo "Please run gen_pages.sh from the ./util directory"
+  exit 1
+fi
+
 ## run this script right from ./utils
-(cd .. && scons docs)
+(cd .. && ./autogen.sh && doxygen)
```
Comment (hazen) [inline util/gen_pages.sh:10]: Do we want to run `autogen.sh` each time we run doxygen? It nukes and repulls `riak_pb`.

Comment (bookshelfdave) [inline util/gen_pages.sh:10]: We shouldn't be publishing docs that often. I don't feel all that strongly about it either way. If you feel as though it should be removed, I'm ok with that.

Comment (bookshelfdave) [inline util/gen_pages.sh:10]: Also, we probably won't be generating the docs site all that often.

Comment (hazen) [inline util/gen_pages.sh:10]: Good point. I was forgetting this is really a utility for us, not for users at large.

**Comment by hazen** [comment]
Cool.  Seems to run and push some stuff to the appropriate branch. :+1:

**Pull request merged**